src/TestFramework/TestFramework/Attributes/TestMethod/IgnoreAttribute.cs

```diff
@@ -26,10 +26,16 @@ public IgnoreAttribute()
     /// <param name="message">
     /// Message specifies reason for ignoring.
     /// </param>
-    public IgnoreAttribute(string? message) => IgnoreMessage = message;
+    public IgnoreAttribute(string? message) => ConditionalIgnoreMessage = message;
 
     /// <summary>
     /// Gets the ignore message indicating the reason for ignoring the test method or test class.
     /// </summary>
-    public string? IgnoreMessage { get; }
+    public string? IgnoreMessage => ConditionalIgnoreMessage;
+
+    public override bool ShouldRun => false;
+
+    public override string? ConditionalIgnoreMessage { get; }
+
+    public override string GroupName => nameof(IgnoreAttribute);
```

Comment:
Shall we skip the `attribute` suffix?

Reply:
Yes. I was only preferring `nameof` rather than literal strings. But doesn't harm as `"Ignore"`